package.json

```diff
@@ -44,9 +45,11 @@
     "zod-form-data": "^1.3.1"
   },
   "devDependencies": {
```

Comment:
Just noticed the weekly job failed because it's missing the `packageManager:` flag here to set the pnpm version 😞   Mind adding that here's while you're at it?

Reply:
Ohhhh, that’s what that was. Yup, will fix!

Reply:
Wait we do have this no?
![image](https://user-images.githubusercontent.com/357379/228008291-3ad91749-c7b5-4e24-b204-c88318172e61.png)

Reply:
Looks like the most recent run maybe failed just because GH was broken?

https://github.com/withastro/astro.build/actions/runs/4532218385/jobs/7983325772#step:6:39